src/test/java/gov/usgs/wma/waterdata/groundwater/DiscreteGroundWaterRulesTest.java

```diff
@@ -23,17 +14,21 @@
 	public void beforeEach() {
 		dgw = new DiscreteGroundWater();
 		dgw.readingQualifiers = "[\"" + LevelStatusCode.ABOVE.getAqDescription() + "\"]";
+		dgw.approvalLevel = "1200";
 	}
 
 
+	//
+	//Rule:  The AQ measurement reading qualifiers (multi-values) are mapped to a single NWISWeb lev_status_cd.
+
 	@Test
-	void singleRecognizedValueIsMapped() throws Exception {
+	void qualifiersSingleRecognizedValueIsMapped() throws Exception {
```

Comment:
I renamed all the qualifier tests to separate them from the approval tests.  In the future, I could see separating individual rules to individual classes, but so far its not too unweildly...